### ISSUE TITLE
Issue when steam name isn't changed and program is closed before changing it

### ISSUE THREAD

**Issue by Skerilyo** [majikat768/HuntStatsLogger#21]
```
Traceback (most recent call last):
  File "main.py", line 26, in <module>
    window = MainWindow()
  File "MainWindow\MainWindow.py", line 11, in __init__
    self.mainframe = MainFrame(self)
  File "MainWindow\MainFrame.py", line 52, in __init__
    self.update()
  File "MainWindow\MainFrame.py", line 72, in update
    self.huntsTab.update()
  File "MainWindow\Hunts\Hunts.py", line 138, in update
    self.updateDetails()
  File "MainWindow\Hunts\Hunts.py", line 133, in updateDetails
    self.teamDetails.update(teams,hunters,hunt,killData)
  File "MainWindow\Hunts\TeamDetails.py", line 119, in update
    if name.lower() == settings.value("steam_name").lower():
AttributeError: 'NoneType' object has no attribute 'lower'
```

**Comment by Skerilyo** [comment]
Sorry the issue is imprecise, a mate of mine had it and I'm trying to reproduce it.

**Comment by Skerilyo** [comment]
Closed by commit bfff7d6db8103e3069302edfc45cfbdc93506f3b
